raft/raft.go

```diff
@@ -509,6 +546,9 @@ func stepLeader(r *raft, m pb.Message) {
 		} else {
 			oldWait := pr.shouldWait()
 			pr.update(m.Index)
+			if r.prs[m.From].maybeSnapshotAbort() {
```

Comment:
probably not.
1) we need to separate the snapshot handling logic
2) we want to print out the log with raft id.